src/components/HorizontalVideo/HorizontalVideo.css

```diff
@@ -0,0 +1,53 @@
+.horizontal-outerbox{
+    display: flex;
+    width: 100%;
+}
+
+.horizontal-img-box{
+    width: 50%;
+    margin-bottom: 0.7rem;
+}
+
+.horizontal-all-detail{
+   margin-left: 0.5rem;
+}
+
+.horizontal-video-des{
+    color: var(--white);
+    font-size: 0.8rem;
+    font-weight: 400;
+    margin-top: 0.3rem;
+}
+
+.horizontal-creator-name{
+    color: var(--grey);
+    font-size: 0.75rem;
+    margin-top: 0.3rem;
```

Comment:
you can use px upto 10px for values less than 1rem

Reply:
ok will use it from now